--- alevt-1.6.1/lang.h.orig	2006-09-02 11:22:48.000000000 +0000
+++ alevt-1.6.1/lang.h	2006-09-02 11:22:53.000000000 +0000
@@ -19,4 +19,4 @@
 void add_enhance(struct enhance *eh, int dcode, u32 *data);
 void enhance(struct enhance *eh, struct vt_page *vtp);
 
-#endif LANG_H
+#endif //LANG_H
--- alevt-1.6.1/xio.c.orig	2006-09-02 11:24:16.000000000 +0000
+++ alevt-1.6.1/xio.c	2006-09-02 11:24:40.000000000 +0000
@@ -31,11 +31,12 @@
 
 static struct dl_head dpys[1];		/* list of all displays */
 
+static void xio_timer();
+static void handle_event(); //forward ref
 
 static int
 timer_init(int argc, char **argv)
 {
-    static void xio_timer();
     int p[2], timer_pid, i;
 
     if (pipe(p) == -1)
@@ -160,7 +161,6 @@
 xio_open_dpy(char *dpy, int argc, char **argv)
 {
     XClassHint classhint[1];
-    static void handle_event(); //forward ref
     struct xio *xio;
 
     if (local_init(argc, argv) == -1)
